.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
```

Comment:
`Focusing on what is best for the community` can be very different from `Focusing on what is best for the project`. Are we sure that the Symfony project should not focus on what's best for the Symfony project?

Reply:
True, it's different, but it's also related. What's good for the project is good for the community and vice versa. I've changed it to `the project and the community`

Reply:
I guess this is a key part of this particular CoC .. the project exists for the community and not vice versa.
As such I would prefer to keep the original version.

Reply:
Someone could be an excellent developer (good for the project) but be extreme rude against woman (bad for the community). _History itself has a great number of examples about this actually._